win32ss/gdi/ntgdi/freetype.c

```diff
@@ -6925,101 +6915,113 @@ NtGdiGetGlyphIndicesW(
     {
         return GDI_ERROR;
     }
-
     FontGDI = ObjToGDI(TextObj->Font, FONT);
     TEXTOBJ_UnlockText(TextObj);
 
-    Buffer = ExAllocatePoolWithTag(PagedPool, cwc*sizeof(WORD), GDITAG_TEXT);
-    if (!Buffer)
+    do
```

Comment:
Please use "goto Exit" instead of this "loop".

Reply:
OK.